content/en/docs/instrumentation/net/automatic/custom.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Create custom traces and metrics
+linkTitle: Custom instrumentation
+description: Custom traces and metrics using .NET automatic instrumentation.
+weight: 30
+---
+
+The automatic instrumentation provides a base you can build on by adding your
+own manual instrumentation. By using both automatic and manual instrumentation,
+you can better instrument the logic and functionality of your applications,
+clients, and frameworks.
```

Comment:
This is correct, but I think what's worth calling out explicitly is that autoinstrumentation configures a tracerprovider and meterprovider for you, so you can skip a lot of the setup in the manual instrumentation link.